NUnitTests/Lessons/21/EnumHomework.cs

```diff
@@ -46,16 +45,24 @@ public void NumberOfIntCorrespondsToSomeTestDataAgeValue()
     public void StringlEmentsArePresentInEnum(string[] list, int expectedNumberPresent, int expectedNumberExtra, bool areAllPresentExpected, bool areExtraElementsExpected)
     {
         var listOfString = list.ToList();
+        var enumValues = Enum.GetNames(typeof(TestDataAge));
 
-        var numberOfStringsWhichPresentInEnum = 
-        var numberOfStringsWhichAreNotPresentInEnum = 
-        var areAllPresent = 
-        var areExtraElements = 
+        // Calculate the number of strings present in the enum
+        var numberOfStringsWhichPresentInEnum = listOfString.Count(str => enumValues.Contains(str));
+        // Calculate the number of strings not present in the enum
+        var numberOfStringsWhichAreNotPresentInEnum = listOfString.Count(str => !enumValues.Contains(str));
+        // Determine if all strings are present in the enum
+        var areAllPresent = numberOfStringsWhichAreNotPresentInEnum == 0;
```

Comment:
this is LINQ task. Please do this according to description "Use such LINQ as Any(), Count(), Contains()"

Reply:
fixed please check

Reply:
thanks, closed